.github/workflows/pull-request.yml

```
@@ -94,6 +94,71 @@
+    if: github.ref == 'refs/heads/develop-postgres'
+    runs-on: ubuntu-latest
+    environment: TALAWA_ENVIRONMENT
+
+    steps:
+      # Step 1: Check out the repository
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      # Step 2: Set up Java
+      - name: Set up Java
+        uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)